### PR TITLE
fix: use different value from values array and check for nil

### PR DESCRIPTION
**Pull request by SMillerDev**
Should resolve the issue in https://github.com/orgs/Homebrew/discussions/4675

### CODE REVIEW

File: cmd/formula-analytics.rb [hunk 1]
```diff
@@ -171,6 +171,8 @@ def influx_analytics(args)
       api_result["results"].first["series"].each do |result|
         next unless result.key? "tags"
 
+        # odebug result
+
```
Comment (MikeMcQuaid) [inline cmd/formula-analytics.rb:175]:
```suggestion
```

File: cmd/formula-analytics.rb [hunk 2]
```diff
@@ -209,7 +211,12 @@ def influx_analytics(args)
 
         # we want the first count out of:
         # "time", "count_options", "count_os_name_and_version", "count_package", "count_tap_name", "count_version"
-        count = result["values"][0][2].to_i
+        count = begin
+          Integer(result["values"][0].last)
```
Comment (MikeMcQuaid) [inline cmd/formula-analytics.rb:215]:
```suggestion
          Integer(result["values"].first.last, 10)
```

File: cmd/formula-analytics.rb [hunk 3]
```diff
@@ -209,7 +211,12 @@ def influx_analytics(args)
...
+        rescue
```
Comment (MikeMcQuaid) [inline cmd/formula-analytics.rb:216]:
```suggestion
        rescue ArgumentError, TypeError
```

File: cmd/formula-analytics.rb [hunk 4]
```diff
@@ -209,7 +211,12 @@ def influx_analytics(args)
 
         # we want the first count out of:
```
Comment (SMillerDev) [inline cmd/formula-analytics.rb:212]:
```suggestion
        # we want any valid count out of:
```

**Review by MikeMcQuaid — APPROVED** [review]
👍🏻 🚢 once suggestions applied

**Pull request merged**